pom.xml

```diff
@@ -32,7 +32,7 @@
     </developers>
 
     <properties>
-        <revision>3.1.3</revision> <!-- CHANGE THIS to adjust project version-->
```

Comment:
make this consistent with other POM's . These versions should go together , if this is 3.1.4 then make it consistent . If it is 3.1.5 make it consistent.